### PR TITLE
Changed width in temp plot

### PR DESCRIPTION
**Pull request by okracht**
Changed the width of the bars in the temperature plot from 30 to 40.

### CODE REVIEW

**Review by Lukeg75 — COMMENTED** [review]
I think these changes make sense.

File: src/plotting.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def read_data(filename,delimiter=',',starting_row=0):
 
 # Create a figure of the processed data
 temperature_figure = plt.figure()
-temperature_plot = plt.bar (processed_temperature_data[:,0],processed_temperature_data[:,2], width=30, color='green')
+temperature_plot = plt.bar (processed_temperature_data[:,0],processed_temperature_data[:,2], width=40, color='green')
```
Comment (Frederick-LaCombe) [inline src/plotting.py:30]: good changes.

Comment (katiebristol) [inline src/plotting.py:30]: This change is great.

**Review by Frederick-LaCombe — APPROVED** [review]
I think these changes are reasonable.

File: src/plotting.py (hunk 1 above)
Comment (AyuniInaMohamaad) [inline src/plotting.py:30]: The width change from '30' to '40' is great.

Comment (DanielAstudilloM) [inline src/plotting.py:30]: why not 50? Nice change :)

**Review by DanielAstudilloM — APPROVED** [review]
Nice changes but I think we can go wider

**Review by AyuniInaMohamaad — APPROVED** [review]
I think the change made to the width of the plot is good.

**Review by katiebristol — APPROVED** [review]
I think these changes are reasonable. Thanks.

**Pull request merged**